### ISSUE TITLE
Storing strain for wildtype in metagenotypes

### ISSUE THREAD

**Issue by barnabynorman** [pombase/canto#1695]
I have now changed the wildtype section of the metagenotype creation page to list the available strains in the session:

![image](https://user-images.githubusercontent.com/1710907/48858310-e95dcb80-edb2-11e8-8b42-7ab6c7579bd4.png)

The code now needs to store the strain.

@kimrutherford is there a way to do this at the moment or can you write it please?

The method that saves the metagenotype is called createMetaGenotype()

The strain is available there and its taxon id is used to create the metagenotype so it just needs something in-between to save the strain name or strain id

**Comment by barnabynorman** [comment]
@kimrutherford it now passes the strain name when saving. Here is an example:

```json
{pathogen_genotype_id: 1, host_taxon_id: 4565, strain_name: "Chinese Spring"}
```

The code passes this object to the Metagenotype service:

```
Metagenotype.create(data);
```

**Comment by kimrutherford** [comment]
Thanks @barnabynorman 

I'll make the server-side changes.

**Comment by jseager7** [comment]
@kimrutherford Unfortunately I'm not able to test this at the moment since I'm still having issues with host strains not being loaded. I'm going to try to reset Canto's data directory and see if that resolves it.

**Comment by jseager7** [comment]
OK, I've tested this now, and it seems to work as expected. I created a metagenotype with the following inputs:

![wt_mg_input](https://user-images.githubusercontent.com/37659591/49870553-a1f3b980-fe0b-11e8-8336-85042282aba2.PNG)

And it successfully created a metagenotype with a host genotype of 'wild type':

![wt_mg_output](https://user-images.githubusercontent.com/37659591/49870637-e54e2800-fe0b-11e8-8c28-b050c662b9b0.PNG)

The annotation table displays this as well:

![wt_mg_annotation](https://user-images.githubusercontent.com/37659591/49870655-ef702680-fe0b-11e8-9772-ce97b629932f.PNG)

- - -

Also, I found out why I was having issues loading the host strains: my loading script on my local development copy wasn't running commands in the same way as the server. I was chaining the organism and strain loading commands together like this:
```
./canto/script/canto_docker /canto/script/canto_load.pl \
--organisms /import_export/host_species.csv \
--organisms /import_export/pathogen_species.csv \
--strains /import_export/host_strains.csv \
--strains /import_export/pathogen_strains.csv
```
...but this didn't work as expected; instead Canto only seemed to load the last file in each sequence (i.e. `pathogen_species.csv` and `pathogen_strains.csv`). Running the commands individually (as the server does) avoids this problem:

```
./canto/script/canto_docker ./script/canto_load.pl --organisms host_species.csv
./canto/script/canto_docker ./script/canto_load.pl --strains host_strains.csv
./canto/script/canto_docker ./script/canto_load.pl --organisms pathogen_species.csv
./canto/script/canto_docker ./script/canto_load.pl --strains pathogen_strains.csv
```
@kimrutherford Is this intended behaviour? That is, was it intended that `canto_load.pl` should support multiple arguments in this way for organisms and strains? Using multiple `--ontology` arguments seems to work fine, so I'm surprised that this doesn't.
